src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-710.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2021-05-28'
 version: 7.1.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-7.1.0.gem'
+features: ["Browser agent can now run on applications using CSP nonces", "Distributed tracing may now be set in server-side application configuration"]
```

Comment:
bugs: Removed MD5 use in the SQL sampler to allow the agent to run in FIPS-compliant environments

Reply:
Sinatra fix

Reply:
Resque fork_per_job

Reply:
Ruby 2 keywords